docs/website/02_examples/incompressibleRubberBlock.md

```diff
@@ -8,21 +8,22 @@ for such a system is defined in the `Solid struct` by the function
 `calculateScalarImpl(const FERequirementType &par, const Eigen::VectorX<ScalarType> &dx)`.
 This function uses the principles of automatic differentiation to provide the stiffness matrix and
 other necessary quantities to perform a static structural analysis.
+It inherits from `FEBase` which provides information about the `localView` of the element.
 
 ## Code highlights
 
-The `struct` named `Solid` is created which is not inherited from any class. It is constructed as shown below:
+The `struct` named `Solid` is created. It is constructed as shown below:
 
 ```cpp
-Solid(const Basis &basis, const typename LocalView::Element &element, double emod, double nu)
-    : localView_{basis.localView()}, emod_{emod}, nu_{nu} {
-  localView_.bind(element);
-  mu_       = emod_ / (2 * (1 + nu_));
-  lambdaMat = convertLameConstants({.emodul = emod_, .nu = nu_}).toLamesFirstParameter();
+Solid(const BasisHandler &basis, const typename LocalView::Element &element, double emod, double nu)
+      : Base(basis, element), emod_{emod}, nu_{nu} {
+    this->localView().bind(element);
```

Comment:
Actually This bind command is not needed since the bind happens in the base class anyway.